### DISCUSSION TITLE
Replace Cookie handling with middleware

### DISCUSSION THREAD

**Discussion by WolfgangFahl** [elimintz/justpy#529] [Ideas]
The signed cookie handling is a standard infrastructure problem that we might want to use from a library instead of doing
the current "roll your own cookie handling". Unfortunately https://fastapi.tiangolo.com/advanced/response-cookies/ doesn't linke to any libraries

We need to find a proper library for this. Candidates might be:
* https://github.com/klen/asgi-sessions
* https://github.com/thearchitector/starlette-securecookies
* https://asgi-signing-middleware.hackan.net/en/stable/cookie/

**Comment by elimintz** [comment]
The session handling in justpy is simple and secure. The user does not need to delve into the details of using secure cookies.
Otherwise, cookies are being handled by starlette. We could take out the session handling logic perhaps and create a starlette middleware https://www.starlette.io/middleware/
